remove_author

sayaikegawa opened the pull request:
Hi @jstac , the author name I deleted by the old PR still stays, so I just deleted it again.

mmcky commented:
@sayaikegawa did you fix this in `lecture-source-py`? Just wondering if the associated `lecture_specific` files are the latest if this hadn't transferred across.

sayaikegawa commented:
Hi @mmcky , I did in [here](https://github.com/QuantEcon/lecture-source-py/pull/960/commits/2d76ea59462fdda981003985f358a4f69f77e4a8)
This is screen shot of a part of this PR. <img width="984" alt="Screen Shot 2020-04-11 at 4 09 03 pm" src="https://user-images.githubusercontent.com/44494439/79036797-fa5ce600-7c0e-11ea-8cfb-d600bc4acd3e.png">

And interestingly, although "Markov Perfect Equilibrium" and "Robust Markov Perfect Equilibrium" use this same code(source/_static/lecture_specific/markov_perf/duopoly_mpe.py) but they are now not in same repo, "Markov Perfect Equilibrium" uses the old one, but "Robust Markov Perfect Equilibrium" uses the latest one. 

I mean,
If you go to pdf of "Markov Perfect Equilibrium", 
<img width="961" alt="Screen Shot 2020-04-11 at 4 14 07 pm" src="https://user-images.githubusercontent.com/44494439/79036861-a8689000-7c0f-11ea-8c5f-f92cfca2d256.png">
(Now the author name is removed in rst by this PR)

and in "Robust Markov Perfect Equilibrium", 
<img width="938" alt="Screen Shot 2020-04-11 at 4 14 28 pm" src="https://user-images.githubusercontent.com/44494439/79036866-bdddba00-7c0f-11ea-89ea-0fd747942e58.png">

And I also removed author name in this [PR](https://github.com/QuantEcon/lecture-python-advanced/pull/111) recently , which I also removed before.

mmcky commented:
hmm -- perhaps we should do a diff of each `lecture_specific` folder to see if they match the `lecture-source-py` set of files.
